client/pubspec.yaml

```diff
@@ -64,6 +64,10 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
 
+dependency_overrides:
+  geolocator: ^13.0.1
+  file_picker: ^8.1.1
```

Comment:
Why do we need file picker here?

Reply:
We don't, I was in middle of PRs with file_picker and record_web to get them to support web-1.0.0, otherwise there was a circular dependency that forced geolocator to old version without using web: ^1.0.0. Which is basically what I had in geolocator and file_picker's place in pubspec.yaml till it was resolved. 

However even after resolving those 2 this week, I noticed geolocator still going to old version. So I forced it to latest version knowing be no more issues with "web", I removed that override. However after cleaning pub cache, and redownloading it, I noticed 2 versions of web downloaded, new version and old version. So there is still a dependency somewhere I haven't been able to locate. So I decided to force geolocator and file_picker to latest versions to show web override was no longer needed.

Reply:
I think we can safely move to `web: ^1.0.0`, no?

Reply:
I believe so, but there is something else downloading older version of web, even with a web override. Not sure if that is the cause of packages not being upgraded. Geolocator will need an override as the "websettings" config in dart file only work with web 1.0.0 till can find cause of them not getting to latest versions.

Reply:
lol I think I see issue on production at least, I did not look further under dev overrides, there is an if block forcing the geolocator version. file_picker also upgrades fine, so I think removing web override, file_picker and that If block would be the solution on production at least. Why dev environment is not upgrading it remains a mystery atm.

Reply:
Nope, removing override on production puts it at geolocator 11.1.0, so its gonna have to be forced, file_picker makes it to latest version on production, so I'll update PR on flet-build-template.

Reply:
Ok i just removed filepicker from client/pubspec.yaml to resolve this so PR can continue :)